hack/module.sh

```diff
@@ -62,7 +58,10 @@ function main() {
   'lint' | 'linter' | 'linters')
     golangci-lint --version
     echo
-    with_modules 'lint_cmd'
+    golangci-lint run --modules-download-mode=readonly --timeout=30m
```

Comment:
This isn't looping over the modules anymore. Do we still care about multiple modules? Getting rid of that consistently across tooling would be a bigger change though.

Reply:
I'm not sure that we do care about multiple modules at this time - should we?

Reply:
We decided to remove the `with_modules` helper.

Reply:
we removed `with_modules` since there's really only one module now